Stripe/STPPaymentMethodCard.m

```diff
@@ -74,4 +74,27 @@ + (nullable instancetype)decodedObjectFromAPIResponse:(nullable NSDictionary *)r
     return card;
 }
 
+#pragma mark - STPCardBrand
+
++ (NSString *)stringFromBrand:(STPCardBrand)brand {
```

Comment:
we should really have this shared between STPPaymentMethodCard and STPCard (also we should probably rename the latter to STPSourceCard). Mind filing a followup?

Reply:
https://jira.corp.stripe.com/browse/IOS-1226